cogs/helpers.py

```diff
@@ -543,6 +544,19 @@ def __init__(self, bot):
         embed.add_field(name="`Food spotted`", value=message)
         await channel.send(embed=embed)
 
+    @commands.command()
+    async def choose(self, ctx, *, inputOpts: str):
+        """Randomly chooses one of the given options delimited by semicola.
+        Usage: ?choose opt1;opt2 
+        """
+        opts = inputOpts.split(';')
+        sel = random.randint(0, (len(opts) - 1))
+        msg = "🤔\n" + opts[sel]
+        embed = discord.Embed(colour=discord.Colour(random.randint(
+            0, 16777215)),
+                              description=msg)
```

Comment:
I'm not a fan of the weird line breakage here – YAPF did this? Could you try fixing it up a bit? I'll approve it anyway, it's up to you if you want to make more commits to this PR.